pyatlan/client/asset.py

```diff
@@ -280,7 +280,9 @@ def find_personas_by_name(
             attributes=attributes,
         )
         results = self.search(search_request)
-        return [asset for asset in results if isinstance(asset, Persona)]
+        if personas := [asset for asset in results if isinstance(asset, Persona)]:
```

Comment:
More curiosity than necessity, but are we not able to reuse the `_search_for_asset_with_name()` passing e.g. `Connection` / `Persona` / `Purpose` as the type and letting it handle the rest (i.e. not repeating code in all of these to run the search, list-comprehend the results, etc)?

Reply:
Sure, we can reuse `_search_for_asset_with_name()` method.